packages/ksf-subscription/src/KSF/Subscription/Component.purs

```diff
@@ -115,36 +114,56 @@ render self@{ props: props@{ subscription: { package } } } =
                      <> (foldMap (showPausedDates <<< filterExpiredPausePeriods) $ self.state.pausedSubscriptions)
                  }
                ]
+               <> deliveryAddress
+               <> foldMap pendingAddressChanges self.state.pendingAddressChanges
                <> foldMap billingDateTerm nextBillingDate
            })
       (if package.digitalOnly
        then mempty
-       else pauseSubscription)
+       else subscriptionUpdates)
       $ Just { extraClasses: [ "subscription--container" ] }
   where
-    billingDateTerm date =
-      [ { term: "Nästa faktureringsdatum:"
-        , descriptions: [ date ]
-        }
-      ]
+    deliveryAddress =
+       if package.digitalOnly
+       then mempty
+       else Array.singleton
+              { term: "Leveransadress:"
+              , descriptions: [ currentDeliveryAddress ]
+              }
+
+    pendingAddressChanges :: Array Persona.PendingAddressChange -> Array DescriptionList.Definition
+    pendingAddressChanges pendingChanges = Array.singleton $
+      { term: "Tillfällig  adressändringar:"
```

Comment:
Should be either both words in plural form
"Tillfälliga adressändringar"
or singular form
"Tillfällig adressändring"